### PR TITLE
Kafka doc updates

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* Jira (Issue Tracker): No Jira

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/kafka-producer/enable-kafka.adoc [hunk 1]
```diff
@@ -16,11 +16,11 @@ ssh -p 8101 admin@localhost
...
 ----
 config:edit org.opennms.features.kafka.producer.client
-config:property-set bootstrap.servers kafka-broker-ip:9092 <1>
+config:property-set bootstrap.servers kafka-server-ip:9092 <1>
 config:update
 ----
-<1> Set the Kafka broker IP by replacing the kafka-broker-ip and port you want to connect to.
-    Add multiple hosts in a comma-separated list, for example, `kafka-broker-1:9092,kafka-broker-2:9092`
+<1> Set the Kafka server IP or hostname by replacing the kafka-server-ip and port of an existing Kafka cluster that you want to connect to.
+    Add multiple hosts in a comma-separated list, for example, `kafka-server-1:9092,kafka-server-2:9092`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/kafka-producer/enable-kafka.adoc:23]:
```suggestion
    Add multiple hosts in a comma-separated list; for example, `kafka-server-1:9092,kafka-server-2:9092`
```

**Review by Bonrob2 — COMMENTED** [review]
one minor change.

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Comment by marshallmassengill** [comment]
LGTM2!

**Pull request merged**